lib/ember_cli.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
 module EmberCli
+  ALIASES ||= {
```

Comment:
```suggestion
  ALIASES = {
```

Our code base is littered with `||=` when assigning constants but we actually have no good reason to do that.

Reply:
I think I normally get comments which say the opposite, and ask me to change `=` to `||=` 😅 

But I agree, I can't think of any good reason, so let's try without and see if it triggers any warnings.